federation/pkg/kubefed/init/init.go

```diff
@@ -389,8 +389,6 @@ func createAPIServer(clientset *client.Clientset, namespace, name, image, creden
 		"--etcd-servers=http://localhost:2379",
 		"--service-cluster-ip-range=10.0.0.0/16",
 		"--secure-port=443",
-		"--token-auth-file=/etc/federation/apiserver/known_tokens.csv",
-		"--basic-auth-file=/etc/federation/apiserver/basic_auth.csv",
```

Comment:
Yeah. I have immediate plans to make this work. See the TODO at the top of this file. Also, the code for this is already written in my, now abandoned, PR - #30615 (https://github.com/kubernetes/kubernetes/pull/30615/commits/aac3f0af52a89b5333501b5395b064c3d3e3776f). I just need to copy it over to this package.